shared-data/python/opentrons_shared_data/protocol/models/protocol_schema_v6.py

```diff
@@ -0,0 +1,151 @@
+from pydantic import BaseModel, Field
+from typing import List, Optional, Dict
+from typing_extensions import Literal
+from enum import Enum
+from opentrons_shared_data.labware.labware_definition import LabwareDefinition
+
+
+class CommandAnnotation(BaseModel):
+    commandIds: List[str]
+    annotationType: str
+
+
+class CornerOffsetFromSlot(BaseModel):
+    x: int
+    y: int
+    z: int
+
+
+class Location(BaseModel):
+    slotName: Optional[int] = None
+    moduleId: Optional[str] = None
+
+
+class VolumeByWell(BaseModel):
+    A1: int
+    B1: int
+
+
+class WellLocation(BaseModel):
+    origin: Optional[str] = None
+    offset: Optional[CornerOffsetFromSlot] = None
+
+
+class Params(BaseModel):
+    slotName: Optional[int] = None
+    axes: Optional[List[str]] = None
+    pipetteId: Optional[str] = None
+    mount: Optional[str] = None
+    moduleId: Optional[str] = None
+    location: Optional[Location] = None
+    labwareId: Optional[str] = None
+    displayName: Optional[str] = None
+    liquidId: Optional[str] = None
+    volumeByWell: Optional[VolumeByWell] = None
+    wellName: Optional[str] = None
+    volume: Optional[float] = None
+    flowRate: Optional[float] = None
+    wellLocation: Optional[WellLocation] = None
+    wait: Optional[int] = None
+    minimumZHeight: Optional[int] = None
+    forceDirect: Optional[bool] = None
+    message: Optional[str] = None
+    coordinates: Optional[CornerOffsetFromSlot] = None
+    axis: Optional[str] = None
+    distance: Optional[float] = None
+    positionId: Optional[str] = None
+
```

Comment:
Missing some payload fields

```suggestion
    temperature: Optional[float] = None
    rpm: Optional[float] = None
    engageHeight: Optional[float] = None
    offset: Optional[CornerOffsetToSlot] = None

```